exercises/practice/gigasecond/.meta/config.json

```diff
@@ -1,6 +1,24 @@
 {
   "blurb": "Given a moment, determine the moment that would be after a gigasecond has passed.",
-  "authors": [],
+  "authors": [
+    "rubysolo"
+  ],
+  "contributors": [
+    "andrewsardone",
+    "angelikatyborska",
+    "Billiam",
```

Comment:
My contribution was a whitespace-only change, and should probably not be included here. Thanks though!